examples/managementCli/src/index.ts

```diff
@@ -459,6 +459,18 @@ program
     handleSdkRes(await sdk.management.theme.import(theme));
   });
 
+// *** Audit commands ***
+
+// search
+program
+  .command('audit-search')
+  .description('Search audit trail up to the last 30 days')
+  .argument('<text>', 'Search for the text in all relevant fields')
```

Comment:
FYI argument is a **non** optional 
```
 npm run start -- audit-search

> node-sdk-management-cli-example@1.0.0 start
> ts-node --esm src/index.ts audit-search

error: missing required argument 'text'
```

so if you want to make it optional you can change it to an `option('-t, --text <text>`, ...)`

but this way is good as well